compiler/luci-interpreter/include/luci_interpreter/core/Tensor.h

```diff
@@ -144,7 +145,7 @@ class Tensor
 
   void writeData(const void *data_ptr, size_t data_size);
 
-  void resize(const Shape &new_shape);
+  void resize(const Shape &new_shape, size_t raw_size = 0);
```

Comment:
plz do not use default values to parameters.
add new method `void resize(const Shape &new_shape, size_t raw_size);`